### PR TITLE
Deis v1.6 release post

### PR DESCRIPTION
**Pull request by mboersma**
Feedback requested.

### CODE REVIEW

File: _posts/2015-05-07-deis-1-6-swarm-tech-preview.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+## 1.6.0 Summary
+
+### New Features
+
+- [swarm](http://docs.deis.io/en/latest/customizing_deis/choosing-a-scheduler/#swarm-scheduler)
+  scheduler is available for preview as an alternative to
+  [fleet](http://docs.deis.io/en/latest/customizing_deis/choosing-a-scheduler/#fleet-scheduler)
+- EC2 provisioning supports the
+  [Proxy Protocol](http://docs.aws.amazon.com/ElasticLoadBalancing/latest/DeveloperGuide/enable-proxy-protocol.html),
+  allowing WebSockets-based apps
```
Comment (bacongobbler) [inline _posts/2015-05-07-deis-1-6-swarm-tech-preview.md]: maybe change this to "allowing websocket support for apps through Amazon's Elastic Load Balancer"

File: _posts/2015-05-07-deis-1-6-swarm-tech-preview.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+- SSL certificates in SAN format are now supported
+- `deis users:list` allows an admin to see all registered Deis users
+- `deis ps:restart` stops and starts all containers of a type, or a single container
```
Comment (bacongobbler) [inline _posts/2015-05-07-deis-1-6-swarm-tech-preview.md]: it also allows you to restart all containers:

```
deis ps:restart # restarts all processes
deis ps:restart web # restarts all web processes
deis ps:restart web.1 # restarts web.1
```

We should clarify this in the client help as well.

Comment (bacongobbler) [inline _posts/2015-05-07-deis-1-6-swarm-tech-preview.md]: filed deis/deis#3639

File: _posts/2015-05-07-deis-1-6-swarm-tech-preview.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
+---
+layout: post
+title: "Deis v1.6 - Swarm Tech Preview"
```
Comment (carmstrong) [inline _posts/2015-05-07-deis-1-6-swarm-tech-preview.md]: We should use "Docker Swarm" here and in the description.

File: _posts/2015-05-07-deis-1-6-swarm-tech-preview.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+Deis is an open source PaaS that makes it easy to deploy and manage applications on your own
+servers. Deis builds upon [Docker](http://docker.io/) and [CoreOS](https://coreos.com/) to provide
+a lightweight PaaS with a Heroku-inspired workflow.
+
+## 1.6.0 Summary
+
+### New Features
+
+- [swarm](http://docs.deis.io/en/latest/customizing_deis/choosing-a-scheduler/#swarm-scheduler)
```
Comment (carmstrong) [inline _posts/2015-05-07-deis-1-6-swarm-tech-preview.md]: Docker Swarm again here

File: _posts/2015-05-07-deis-1-6-docker-swarm-tech-preview.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+### Under the Hood
+
+- bumped CoreOS to 633.1.0
+- updated Ceph to "Hammer" release
+- bumped `confd` to v0.9.0 in all components
+- builder now uses the official `heroku/cedar:14` image
+- router updated to nginx 1.9 for stream module TCP support
+- added new C4 instance types to EC2 provisioning
+- added sfo1 and ams2 data centers to DigitalOcean provisioning
+- the Deis project is now supported by [Engine Yard](https://www.engineyard.com/engine-yard-and-deis)!
```
Comment (carmstrong) [inline _posts/2015-05-07-deis-1-6-docker-swarm-tech-preview.md]: We may want to promote this to before the <more> tag ;)

File: _posts/2015-05-07-deis-1-6-docker-swarm-tech-preview.md [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+- @nathansamson: Non existing app returns http 200, After upgrading to 1.5.1 deis uses lot more resources, Application SSL is not working when using multiple certs, Deis Builder resets key after upgrade / restart
+- @ngpestelos: Unable to push a new app in vagrant, HOST_IPADDR: parameter null or not set, deis-database: create_bucket failed after upgrading platform, deis-store-metadata won't start (missing ceph-mds), logs.papertrailapp.com drain type is not implemented, Unable to login to controller after upgrading to 1.6.0-dev, Unable to build 1.6.0-dev
+- @olalonde: /usr/bin/dock cannot allocate memory
+- @onbjerg: Post http://*:8000/v1/hooks/build: EOF, /home/git/project.git/hooks/pre-receive: No such file or directory, feat(client): add optional path to config:push
```
Comment (carmstrong) [inline _posts/2015-05-07-deis-1-6-docker-swarm-tech-preview.md]: Does this line actually break the rest of the markup, or is that just a GitHub styling issue?

Comment (mboersma) [inline _posts/2015-05-07-deis-1-6-docker-swarm-tech-preview.md]: It's just a GitHub styling issue AFAICTL (which also happens in Atom, interestingly). But I've been testing this locally with `bundle exec jekyll serve --baseurl=''` and it renders fine.

Comment (carmstrong) [inline _posts/2015-05-07-deis-1-6-docker-swarm-tech-preview.md]: SGTM

File: _posts/2015-05-07-deis-1-6-docker-swarm-tech-preview.md [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+
+### New Container Schedulers
+
+Application containers require a resource-aware scheduler to improve cluster efficiency at scale
+and to alleviate issues with [cluster balancing after the loss of a node](https://github.com/deis/deis/issues/2920).
+In addition to the [Swarm](https://github.com/deis/deis/pull/3612) technology preview in this
+release, prototyping is underway with [Kubernetes](https://github.com/deis/deis/issues/2744)
+and [Mesos](https://github.com/deis/deis/issues/3538).
+
+Deis will continue to support [fleet](https://github.com/coreos/fleet) for both control plane and
```
Comment (carmstrong) [inline _posts/2015-05-07-deis-1-6-docker-swarm-tech-preview.md]: Maybe change "support" to "default to" or something to emphasize that we still rely quite heavily on fleet?

**Comment by carmstrong** [comment]
LGTM. Nice work as always, @mboersma

**Comment by bacongobbler** [comment]
LGTM

**Comment by mboersma** [comment]
Thanks! I will leave this as-is until tomorrow a.m. when we plan to do the release, so please add any other comments you think of.

File: _posts/2015-05-07-deis-1-6-docker-swarm-tech-preview.md [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+servers. Deis builds upon [Docker](http://docker.io/) and [CoreOS](https://coreos.com/) to provide
+a lightweight PaaS with a Heroku-inspired workflow.
+
+## 1.6.0 Summary
+
+### New Features
+
+- Docker [Swarm](http://docs.deis.io/en/latest/customizing_deis/choosing-a-scheduler/#swarm-scheduler)
+  scheduler is available for preview as an alternative to
+  [Fleet](http://docs.deis.io/en/latest/customizing_deis/choosing-a-scheduler/#fleet-scheduler)
```
Comment (carmstrong) [inline _posts/2015-05-07-deis-1-6-docker-swarm-tech-preview.md]: Maybe we should clarify "...as an alternative to fleet for scheduling application units"? Perhaps too verbose for release notes, but it's more correct.

**Pull request merged**